### DISCUSSION TITLE
SSL Certificate

### DISCUSSION THREAD

**Discussion by AnythingMayBe** [pockethost/pockethost#199] [Q&A]
Hi, the SSL certificate of the [https://pockethost.io/](https://pockethost.io/) website is gonna expire in three days, the Friday, 24 Mar 2023 13:18:27 GMT, are anyone aware of it?

**Comment by benallfree** [comment]
@AnythingMayBe Thank you! I believe it's set to auto-renew, but I'll check to be sure.

**Comment by AnythingMayBe** [comment]
Okay, nice!

**Comment by benallfree** [comment]
Completed
